pages/features/list.md

```diff
@@ -8,7 +8,7 @@ General and Miscellaneous features.
 
 ### Mod Checker
 
```

Comment:
Hi so can you add the link to the feature just like line 20 of custom main menu?

Reply:
![image](https://github.com/PartlySaneStudios/partly-sane-skies/assets/106500506/4aa65451-817b-4ca0-a4d6-73ddebbc40e3)

Like this?

Reply:
Yep

Reply:
![image](https://github.com/PartlySaneStudios/partly-sane-skies/assets/106500506/c1dbd3db-49c3-4cc1-9644-20131170f60e)
It gives me an error, can I assume its working and just ignore it?

Reply:
Do general/custom_main_menu.md

Reply:
"Cannot resolve file: 'General'"

Reply:
Send snippet

Reply:
![image](https://github.com/PartlySaneStudios/partly-sane-skies/assets/106500506/c107cdd4-0e33-4f24-988e-37e5117a54b7)

Reply:
Ohhhh I see. For the readme, do pages/features/genera/custom_main_menu.md, and for the list.md do general/custom_main_menu.md

Reply:
Original requests was to put the link to the page for all of the pages you created